library/std/src/collections/hash/set.rs

```diff
@@ -1536,6 +1552,13 @@ impl<K: fmt::Debug> fmt::Debug for Iter<'_, K> {
     }
 }
 
+#[stable(feature = "clone_iters_hash", since = "CURRENT_RUSTC_VERSION")]
+impl<K> Clone for IntoIter<K> {
```

Comment:
```suggestion
impl<K> Clone for IntoIter<K>
where
    K: Clone,
{
```

Reply:
Nice catch. I was mostly holding off fixing the errors before rust-lang/hashbrown#543 is merged, but I had missed this one.